cluster/charts/rook-ceph/templates/role.yaml

```diff
@@ -43,6 +43,18 @@ rules:
   - network-attachment-definitions
   verbs:
   - get
+- apiGroups:
```

Comment:
Shouldn't we have a condition on a value like `enableMonitoring` and then add these?

Reply:
It depends on cephcluster manifest, not on operator helm release.
In my opinion, if operator can require those operations it should have access to do so.